modules/oe_whitelabel_news/templates/node--oe-news--full.html.twig

```diff
@@ -4,17 +4,24 @@
  * News full display.
  */
 #}
+
+{% if content.oe_publication_date|render is not empty %}
+  {% set _meta = [] %}
+  {% set _news_publication_date %}
+    {{ content.oe_publication_date|field_value }}
+  {% endset %}
+  {% set _meta = _meta|merge([content.oe_publication_date]) %}
+{% endif %}
+
 <article{{attributes}}>
   {{ pattern('content_banner', {
     background: 'gray',
     title: label,
     content: content.oe_summary,
     image: image,
-    meta: [
-      content.oe_publication_date,
-    ]
+    meta: _meta,
```

Comment:
there is the same problem here as we had for events
<img width="264" alt="Screenshot 2022-02-28 at 17 31 28" src="https://user-images.githubusercontent.com/11507647/156013258-78588d33-f608-4cb5-8a3e-24270cb0d7db.png">
so lets add the fix, also mention somewhere here in a comment that this is to be removed once OEL-1227 is implemented, same for the events one
```suggestion
    meta: _meta,
    
    attributes: create_attribute().addClass(['ps-0']),
```
I also see the content is not in align with the title, not sure what these changes fix..